official/utils/accelerator/tpu.py

```diff
@@ -14,13 +14,9 @@
 # ==============================================================================
 """Functions specific to running TensorFlow on TPUs."""
 
-import time
-
 import tensorflow as tf
-from tensorflow.python.framework import dtypes
 from tensorflow.python.framework import ops
```

Comment:
Just curious, is there a reason the dtypes and ops are directly imported? (e.g. array_ops.reshape instead of tf.reshape)

Reply:
No! Looks like everything can be done with `tf.*` ops. Thanks for pointing this out.